client/src/assets/thumbnails.json

```diff
@@ -3,5 +3,9 @@
   "leagueoflegends": "https://mir-s3-cdn-cf.behance.net/projects/404/6ac6bf42470215.Y3JvcCw5MjIsNzIyLDE4MCww.png",
   "overwatch": "https://uploads-community-endlessos-com.s3.dualstack.us-west-2.amazonaws.com/optimized/2X/0/03d66e753906d7dd513b9d892c97beb8753383b6_2_690x388.jpeg",
   "csgo": "https://steamuserimages-a.akamaihd.net/ugc/480021807334219792/15EDCA029E5F7974784EA65F131745EB256756DB/",
-  "minecraft": "https://wearvr-static.global.ssl.fastly.net/thumbnails/images/000/002/056/thumbnail/Minecraft-Wallpaper-100.jpg?1488896689"
+  "minecraft": "https://wearvr-static.global.ssl.fastly.net/thumbnails/images/000/002/056/thumbnail/Minecraft-Wallpaper-100.jpg?1488896689",
+  "pubg": "https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.velosofy.com%2Ftemplate%2Fpubg-yt-template-6jud15qhk6u&psig=AOvVaw0EojPj2u4zjX0sy7nipz3k&ust=1586991190316000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCPDNuYSB6egCFQAAAAAdAAAAABAD",
```

Comment:
I haven't been able to get any of the new images to work. From looking at the url's they're a bit strange, and just redirect to youtube which is probably why.

On closer inspection I'm getting this error in the console.
![image](https://user-images.githubusercontent.com/25234283/79364541-b61b5f80-7f06-11ea-8bb8-94f425eef807.png)

Reply:
Oops I copied the wrong links. They should be image link not the address link